zkevm-circuits/src/evm_circuit/param.rs

```diff
@@ -64,7 +64,7 @@ pub const FIXED_TABLE_LOOKUPS: usize = 8;
 pub const TX_TABLE_LOOKUPS: usize = 4;
 
 /// Rw Table lookups done in EVMCircuit
-pub const RW_TABLE_LOOKUPS: usize = 8;
```

Comment:
increase 5 columns for rw_lookup to make sure `EndChunk` height to be `1`, same as `EndBlock`

> `EndChunk` consume more rw_lookup than `EndBlock` is because `EndChunk` write StepState fields into `rw_table`